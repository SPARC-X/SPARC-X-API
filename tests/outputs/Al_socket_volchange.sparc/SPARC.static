***************************************************************************
                        Atom positions (socket step 1)                    
***************************************************************************
Fractional coordinates of Al:
      0.0435568480       0.0098804249       0.0241663700
      0.0553306963       0.5461125430       0.4758696820
      0.5234589733      -0.0037372150       0.4974513864
      0.5101382346       0.5035566314       0.0359079878
Lattice (Bohr):
  7.6533908096E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.6533908096E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.6533908096E+00 
Total free energy (Ha): -9.041897494954259E+00
Atomic forces (Ha/Bohr):
 -6.3263660275E-03   4.1224438525E-03  -9.7573141360E-03
 -1.1198940824E-02  -9.4391851504E-03   1.3724648200E-02
  7.1551844744E-03   6.8108725330E-03   8.1322757442E-03
  1.0370122377E-02  -1.4941312350E-03  -1.2099609809E-02
Stress (GPa): 
 -1.2943266427E+01  -7.3004908329E-01   1.0042725753E+00 
 -7.3004908329E-01  -1.3976465331E+01   5.9432478434E-01 
  1.0042725753E+00   5.9432478434E-01  -8.2525496305E+00
***************************************************************************
                        Atom positions (socket step 2)                    
***************************************************************************
Fractional coordinates of Al:
      0.0401072929      -0.0151050966      -0.0130412778
     -0.0264930524       0.5213680896       0.4431718840
      0.5430817720      -0.0187952321       0.5078775085
      0.4938427068       0.5361014305      -0.0508676718
Lattice (Bohr):
  7.7299247177E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.7299247177E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.7299247177E+00 
Total free energy (Ha): -9.043789471327491E+00
Atomic forces (Ha/Bohr):
 -4.5160329928E-03   1.0836654669E-02  -2.0034847850E-03
  7.8472069094E-03  -1.0356602905E-02   4.6601556005E-03
 -5.6452194926E-03   1.4120642271E-02  -9.1708495469E-03
  2.3140455760E-03  -1.4600694035E-02   6.5141787315E-03
Stress (GPa): 
 -7.3542604262E+00   1.8630500028E+00  -7.3044231388E-01 
  1.8630500028E+00  -2.2936813817E+01   1.6489555596E+00 
 -7.3044231388E-01   1.6489555596E+00  -9.3987769886E-01
***************************************************************************
                        Atom positions (socket step 3)                    
***************************************************************************
Fractional coordinates of Al:
     -0.0102903172      -0.0013893044      -0.0527455826
      0.0405005138       0.4557176399       0.4792161144
      0.5124168251      -0.0307478540       0.4738777230
      0.4775553675       0.5136161492       0.0565977287
Lattice (Bohr):
  7.8064586258E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.8064586258E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.8064586258E+00 
Total free energy (Ha): -9.049938894899322E+00
Atomic forces (Ha/Bohr):
  3.6109364340E-03  -2.6690542034E-03  -4.0562261364E-03
 -1.3775701683E-02   4.3698977978E-03   5.4623877892E-03
  2.2823680670E-03   5.1324965010E-03   4.1411554263E-03
  7.8823971824E-03  -6.8333400954E-03  -5.5473170791E-03
Stress (GPa): 
  5.3653952236E+00   5.5029845435E-01   1.4612463857E+00 
  5.5029845435E-01   8.7131123988E-01  -8.1756988882E-01 
  1.4612463857E+00  -8.1756988882E-01  -3.3530577155E+01
***************************************************************************
                        Atom positions (socket step 4)                    
***************************************************************************
Fractional coordinates of Al:
      0.0441636660       0.0107780210       0.0023826535
     -0.0460121655       0.4931509086       0.4912405190
      0.4979570005      -0.0154814982       0.4989180699
      0.4882168388       0.4675588950       0.0218425279
Lattice (Bohr):
  7.8829925339E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.8829925339E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.8829925339E+00 
Total free energy (Ha): -9.055230179659366E+00
Atomic forces (Ha/Bohr):
  4.0067152851E-03  -3.8356803587E-03  -1.5734798624E-04
 -5.8923489168E-03   5.2624518903E-04   2.3814703451E-03
  2.7235026767E-03  -1.5412261635E-03   3.2278071695E-03
 -8.3786904498E-04   4.8506613332E-03  -5.4519295284E-03
Stress (GPa): 
 -2.8943214660E+01  -4.1693591016E-01  -2.6573714866E-01 
 -4.1693591016E-01  -1.2275430836E+00   1.7648242290E-01 
 -2.6573714866E-01   1.7648242290E-01   1.5315611581E+01
***************************************************************************
                        Atom positions (socket step 5)                    
***************************************************************************
Fractional coordinates of Al:
      0.0012484372       0.0123444774      -0.0245903440
      0.0171258891       0.4896715674       0.4608746362
      0.4840072403       0.0147796339       0.5082037045
      0.4716672436       0.5152757948      -0.0021725167
Lattice (Bohr):
  7.9595264420E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.9595264420E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.9595264420E+00 
Total free energy (Ha): -9.054700435144742E+00
Atomic forces (Ha/Bohr):
 -3.5211286475E-03  -9.2550993639E-04  -1.0217768127E-03
 -3.9595343899E-03   2.0838322550E-03  -1.7852101572E-04
  3.4271176949E-03  -1.8989650523E-03   1.4698018221E-03
  4.0535453425E-03   7.4064273374E-04  -2.6950399367E-04
Stress (GPa): 
 -5.0990822082E+00   2.3887198275E-01   4.7911765765E-01 
  2.3887198275E-01   6.8548759897E+00  -2.1589044488E-01 
  4.7911765765E-01  -2.1589044488E-01  -8.4175219309E+00
***************************************************************************
                        Atom positions (socket step 6)                    
***************************************************************************
Fractional coordinates of Al:
      0.0108945058      -0.0081696334       0.0600190417
     -0.0062244970       0.5027064158       0.5390736078
      0.4775498172      -0.0146083125       0.5046321784
      0.4918550628       0.4705490219      -0.0050586793
Lattice (Bohr):
  7.6533908096E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.6533908096E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.6533908096E+00 
Total free energy (Ha): -9.044015698563159E+00
Atomic forces (Ha/Bohr):
 -6.4899505243E-03  -1.7701350735E-03  -3.5854169022E-03
 -2.2501897094E-03  -5.5755156756E-03   4.7195485437E-04
  6.0201216947E-03   1.9367758860E-03   2.1526464607E-03
  2.7200185390E-03   5.4088748630E-03   9.6081558714E-04
Stress (GPa): 
 -1.3875184824E+01  -2.0465820419E-01  -6.5679702138E-01 
 -2.0465820419E-01   1.0203801902E+01  -9.1893504657E-02 
 -6.5679702138E-01  -9.1893504657E-02  -2.9928358694E+01
***************************************************************************
                        Atom positions (socket step 7)                    
***************************************************************************
Fractional coordinates of Al:
     -0.0076983623       0.0180000969       0.0053782911
     -0.0221997974       0.4385980086       0.5225494203
      0.5278287340      -0.0373850180       0.5404763230
      0.4893853431       0.5649698903       0.0148598086
Lattice (Bohr):
  7.5768569015E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.5768569015E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.5768569015E+00 
Total free energy (Ha): -9.035923087337675E+00
Atomic forces (Ha/Bohr):
  5.5234101204E-03   2.7883136233E-03   1.0138002001E-02
  8.5665470167E-03   5.4650245470E-03  -3.2618152821E-03
 -1.5435274056E-02   1.4611065082E-03  -1.3778924569E-02
  1.3453169189E-03  -9.7144446785E-03   6.9027378497E-03
Stress (GPa): 
  1.7755918677E+01  -4.6761884228E-01  -6.3954032952E-01 
 -4.6761884228E-01  -4.4864277040E+01   1.8064780004E+00 
 -6.3954032952E-01   1.8064780004E+00  -2.5395712589E+01
***************************************************************************
                        Atom positions (socket step 8)                    
***************************************************************************
Fractional coordinates of Al:
      0.0417413754      -0.0115046264       0.0008103744
      0.0100621304       0.4805204190       0.5000510018
      0.4999780152      -0.0433265261       0.5251273582
      0.5148271238       0.4845573093      -0.0042357595
Lattice (Bohr):
  7.5003229934E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.5003229934E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.5003229934E+00 
Total free energy (Ha): -9.034213810754752E+00
Atomic forces (Ha/Bohr):
 -1.0203921492E-02  -1.3696717334E-03   3.7408611728E-03
 -9.1493407603E-04  -2.3459252992E-03  -4.6347656751E-04
  7.7987694101E-03   7.8134445048E-03  -8.4685561789E-03
  3.3200861578E-03  -4.0978474722E-03   5.1911715736E-03
Stress (GPa): 
 -1.7029525887E+01  -2.2868816192E-01   5.3988096056E-01 
 -2.2868816192E-01  -1.5856367050E+01   5.1336081411E-01 
  5.3988096056E-01   5.1336081411E-01  -1.6622352739E+01
***************************************************************************
                        Atom positions (socket step 9)                    
***************************************************************************
Fractional coordinates of Al:
      0.0022519683       0.0269452527      -0.0480733410
     -0.0342308526       0.4432965043       0.5595020816
      0.5426626214       0.0544334885       0.5196253738
      0.5241091629       0.4707795766       0.0473176200
Lattice (Bohr):
  7.4237890854E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.4237890854E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.4237890854E+00 
Total free energy (Ha): -8.984187454249179E+00
Atomic forces (Ha/Bohr):
  1.3888173101E-02  -4.4252582143E-02   3.8440095751E-02
  4.3217860585E-02   6.6877499459E-02  -2.6380044419E-02
 -4.4366048349E-02  -4.9619304866E-02  -9.7021312649E-03
 -1.2739985336E-02   2.6994387550E-02  -2.3579200668E-03
Stress (GPa): 
  3.6742100578E+00  -5.3615148319E+00  -1.9878133216E+00 
 -5.3615148319E+00  -4.5825327943E+01   7.4079415847E+00 
 -1.9878133216E+00   7.4079415847E+00  -4.9703402576E+01
***************************************************************************
                        Atom positions (socket step 10)                    
***************************************************************************
Fractional coordinates of Al:
      0.0000273717      -0.0071492363      -0.0275571927
     -0.0003180928       0.4906577416       0.4881201145
      0.4625350326      -0.0121202958       0.4940572697
      0.4840012973       0.5157010143       0.0429658594
Lattice (Bohr):
  7.3472551773E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   7.3472551773E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   7.3472551773E+00 
Total free energy (Ha): -9.013003282724982E+00
Atomic forces (Ha/Bohr):
 -9.8748678836E-03   3.9446542852E-03   5.3614896942E-03
 -1.1482985198E-02   6.4063539325E-04   6.3863125643E-03
  1.4985108306E-02   6.8363718074E-03   4.8060204115E-03
  6.3727447759E-03  -1.1421661486E-02  -1.6553822670E-02
Stress (GPa): 
 -1.8072352208E+01   2.0146885240E-01   1.2059399459E+00 
  2.0146885240E-01  -1.2030264340E+01  -8.9935643428E-01 
  1.2059399459E+00  -8.9935643428E-01  -4.6324288232E+01
