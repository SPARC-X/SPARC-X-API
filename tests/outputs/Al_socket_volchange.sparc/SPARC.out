***************************************************************************
*                       SPARC (version Oct 31, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Thu Jan 18 08:03:34 2024                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 1 1 1 
LATVEC:
7.653390809639560 0.000000000000000 0.000000000000000 
0.000000000000000 7.653390809639560 0.000000000000000 
0.000000000000000 0.000000000000000 7.653390809639560 
FD_GRID: 15 15 15
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.000950043469
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 12
CHEB_DEGREE: 17
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-03
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-05
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-06
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 2.60E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0
MIXING_PARAMETER: 1
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
NUM_CHEFSI: 1
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: SPARC
***************************************************************************
                              Socket Mode                                  
***************************************************************************
SOCKET_HOST: localhost
SOCKET_PORT: 12345
SOCKET_INET: 1
SOCKET_MAX_NITER: 10000
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
7.653390809639560 0.000000000000000 0.000000000000000 
0.000000000000000 7.653390809639560 0.000000000000000 
0.000000000000000 0.000000000000000 7.653390809639560 
Volume: 4.4829270488E+02 (Bohr^3)
Density: 2.4074929800E-01 (amu/Bohr^3), 2.6978060734E+00 (g/cc)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 2
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 1 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  2
Mesh spacing                       :  0.510226 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  SPARC.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  12
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  13_Al_3_1.9_1.9_pbe_n_v1.0.psp8
Atomic mass                        :  26.9815385
Pseudocharge radii of atom type 1  :  7.14 7.14 7.14 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  3.01 MB
Estimated memory per processor     :  1.50 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2624042360E+00        2.019E-01        0.018
2            -2.2643305919E+00        2.444E-01        0.007
3            -2.2606432887E+00        6.854E-02        0.007
4            -2.2606097868E+00        7.455E-02        0.007
5            -2.2605338193E+00        5.258E-02        0.007
6            -2.2604872391E+00        2.541E-02        0.007
7            -2.2604747912E+00        3.460E-03        0.007
8            -2.2604770255E+00        8.484E-03        0.006
9            -2.2604758623E+00        6.046E-03        0.006
10           -2.2604739746E+00        1.915E-03        0.006
11           -2.2604743737E+00        5.961E-05        0.006
Total number of SCF: 11    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2604743737E+00 (Ha/atom)
Total free energy                  : -9.0418974950E+00 (Ha)
Band structure energy              :  5.3650303495E-01 (Ha)
Exchange correlation energy        : -4.3686508563E+00 (Ha)
Self and correction energy         : -1.2498732238E+01 (Ha)
-Entropy*kb*T                      : -3.5815827173E-03 (Ha)
Fermi level                        :  1.2737181165E-01 (Ha)
RMS force                          :  1.5302594623E-02 (Ha/Bohr)
Maximum force                      :  2.0071882326E-02 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           :  1.1724093796E+01 (GPa)
Maximum stress                     :  1.3976465331E+01 (GPa)
Time for stress calculation        :  0.024 (sec)
===================================================================
                    Self Consistent Field (SCF#2)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2995980202E+00        4.247E-01        0.007
2            -2.2640160103E+00        2.194E-01        0.007
3            -2.2624927669E+00        1.627E-01        0.007
4            -2.2612589699E+00        9.871E-02        0.006
5            -2.2611071439E+00        8.378E-02        0.009
6            -2.2609638034E+00        2.776E-02        0.007
7            -2.2609593764E+00        2.570E-02        0.007
8            -2.2609495574E+00        1.130E-02        0.007
9            -2.2609461968E+00        6.206E-03        0.006
10           -2.2609473678E+00        7.382E-04        0.006
Total number of SCF: 10    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2609473678E+00 (Ha/atom)
Total free energy                  : -9.0437894713E+00 (Ha)
Band structure energy              :  4.6705254027E-01 (Ha)
Exchange correlation energy        : -4.3409210683E+00 (Ha)
Self and correction energy         : -1.2498725960E+01 (Ha)
-Entropy*kb*T                      : -3.5197435430E-03 (Ha)
Fermi level                        :  1.2062106575E-01 (Ha)
RMS force                          :  1.4906744366E-02 (Ha/Bohr)
Maximum force                      :  1.7758533799E-02 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           :  1.0410317314E+01 (GPa)
Maximum stress                     :  2.2936813817E+01 (GPa)
Time for stress calculation        :  0.025 (sec)
===================================================================
                    Self Consistent Field (SCF#3)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.3031454833E+00        4.575E-01        0.008
2            -2.2644591631E+00        2.289E-01        0.006
3            -2.2649697671E+00        1.816E-01        0.007
4            -2.2626355483E+00        6.776E-02        0.007
5            -2.2624932164E+00        1.763E-02        0.007
6            -2.2624911274E+00        1.485E-02        0.006
7            -2.2624863392E+00        1.056E-02        0.006
8            -2.2624856009E+00        9.417E-03        0.006
9            -2.2624862012E+00        5.087E-03        0.006
10           -2.2624847237E+00        8.577E-04        0.006
Total number of SCF: 10    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2624847237E+00 (Ha/atom)
Total free energy                  : -9.0499388949E+00 (Ha)
Band structure energy              :  4.0509703704E-01 (Ha)
Exchange correlation energy        : -4.3182978580E+00 (Ha)
Self and correction energy         : -1.2498661550E+01 (Ha)
-Entropy*kb*T                      : -2.5160774120E-03 (Ha)
Fermi level                        :  1.1558582875E-01 (Ha)
RMS force                          :  1.0073737490E-02 (Ha/Bohr)
Maximum force                      :  1.5450037022E-02 (Ha/Bohr)
Time for force calculation         :  0.013 (sec)
Pressure                           :  9.0979568973E+00 (GPa)
Maximum stress                     :  3.3530577155E+01 (GPa)
Time for stress calculation        :  0.024 (sec)
===================================================================
                    Self Consistent Field (SCF#4)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2809052142E+00        3.722E-01        0.008
2            -2.2665856436E+00        2.477E-01        0.006
3            -2.2657167405E+00        1.752E-01        0.006
4            -2.2638432715E+00        3.524E-02        0.007
5            -2.2638525599E+00        4.349E-02        0.007
6            -2.2638070881E+00        9.057E-03        0.006
7            -2.2638080205E+00        6.056E-03        0.006
8            -2.2638082562E+00        2.075E-03        0.006
9            -2.2638081132E+00        1.682E-03        0.006
10           -2.2638084417E+00        1.492E-03        0.006
11           -2.2638075449E+00        8.829E-05        0.006
Total number of SCF: 11    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2638075449E+00 (Ha/atom)
Total free energy                  : -9.0552301797E+00 (Ha)
Band structure energy              :  3.5437186141E-01 (Ha)
Exchange correlation energy        : -4.2791348334E+00 (Ha)
Self and correction energy         : -1.2498805405E+01 (Ha)
-Entropy*kb*T                      : -2.6306989463E-03 (Ha)
Fermi level                        :  1.0712483150E-01 (Ha)
RMS force                          :  5.9418042665E-03 (Ha/Bohr)
Maximum force                      :  7.3453710246E-03 (Ha/Bohr)
Time for force calculation         :  0.013 (sec)
Pressure                           :  4.9517153875E+00 (GPa)
Maximum stress                     :  2.8943214660E+01 (GPa)
Time for stress calculation        :  0.025 (sec)
===================================================================
                    Self Consistent Field (SCF#5)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2763056160E+00        3.321E-01        0.008
2            -2.2675366827E+00        2.266E-01        0.007
3            -2.2644111994E+00        1.408E-01        0.006
4            -2.2641050899E+00        1.133E-01        0.006
5            -2.2637205218E+00        4.382E-02        0.006
6            -2.2636831058E+00        2.142E-02        0.006
7            -2.2636826268E+00        2.029E-02        0.006
8            -2.2636744903E+00        6.556E-03        0.006
9            -2.2636761002E+00        8.172E-03        0.006
10           -2.2636752909E+00        1.427E-03        0.006
11           -2.2636751088E+00        2.524E-04        0.005
Total number of SCF: 11    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2636751088E+00 (Ha/atom)
Total free energy                  : -9.0547004351E+00 (Ha)
Band structure energy              :  2.9578167628E-01 (Ha)
Exchange correlation energy        : -4.2447229426E+00 (Ha)
Self and correction energy         : -1.2498659918E+01 (Ha)
-Entropy*kb*T                      : -3.5202266749E-03 (Ha)
Fermi level                        :  9.8352167765E-02 (Ha)
RMS force                          :  4.1433724264E-03 (Ha/Bohr)
Maximum force                      :  4.4779615010E-03 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           :  2.2205760498E+00 (GPa)
Maximum stress                     :  8.4175219309E+00 (GPa)
Time for stress calculation        :  0.026 (sec)
===================================================================
                    Self Consistent Field (SCF#6)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2701028337E+00        2.997E-01        0.007
2            -2.2619470497E+00        1.694E-01        0.007
3            -2.2626666880E+00        1.825E-01        0.007
4            -2.2610422018E+00        3.796E-02        0.007
5            -2.2610848498E+00        5.947E-02        0.007
6            -2.2610084159E+00        1.388E-02        0.007
7            -2.2610054185E+00        9.355E-03        0.006
8            -2.2610041922E+00        4.065E-03        0.006
9            -2.2610040438E+00        2.050E-03        0.006
10           -2.2610036660E+00        1.261E-03        0.006
11           -2.2610039246E+00        4.797E-05        0.006
Total number of SCF: 11    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2610039246E+00 (Ha/atom)
Total free energy                  : -9.0440156986E+00 (Ha)
Band structure energy              :  5.5281669496E-01 (Ha)
Exchange correlation energy        : -4.3629888711E+00 (Ha)
Self and correction energy         : -1.2498708777E+01 (Ha)
-Entropy*kb*T                      : -3.1889774798E-03 (Ha)
Fermi level                        :  1.2939226158E-01 (Ha)
RMS force                          :  6.6160532297E-03 (Ha/Bohr)
Maximum force                      :  7.6228636580E-03 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           :  1.1199913872E+01 (GPa)
Maximum stress                     :  2.9928358694E+01 (GPa)
Time for stress calculation        :  0.025 (sec)
===================================================================
                    Self Consistent Field (SCF#7)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2872751054E+00        3.987E-01        0.008
2            -2.2607143717E+00        2.240E-01        0.007
3            -2.2621006239E+00        1.993E-01        0.007
4            -2.2592715371E+00        8.911E-02        0.007
5            -2.2589919398E+00        1.499E-02        0.006
6            -2.2589991527E+00        2.806E-02        0.006
7            -2.2589841988E+00        1.321E-02        0.007
8            -2.2589798598E+00        2.854E-03        0.006
9            -2.2589807718E+00        6.342E-04        0.006
Total number of SCF: 9     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2589807718E+00 (Ha/atom)
Total free energy                  : -9.0359230873E+00 (Ha)
Band structure energy              :  6.0205038287E-01 (Ha)
Exchange correlation energy        : -4.4111566119E+00 (Ha)
Self and correction energy         : -1.2498686849E+01 (Ha)
-Entropy*kb*T                      : -2.2194302029E-03 (Ha)
Fermi level                        :  1.3889533790E-01 (Ha)
RMS force                          :  1.3821010737E-02 (Ha/Bohr)
Maximum force                      :  2.0742258307E-02 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           :  1.7501356984E+01 (GPa)
Maximum stress                     :  4.4864277040E+01 (GPa)
Time for stress calculation        :  0.026 (sec)
===================================================================
                    Self Consistent Field (SCF#8)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.2766305756E+00        3.393E-01        0.008
2            -2.2613529280E+00        1.976E-01        0.007
3            -2.2598355756E+00        1.742E-01        0.006
4            -2.2591098252E+00        1.258E-01        0.007
5            -2.2587282319E+00        7.755E-02        0.007
6            -2.2586437660E+00        6.743E-02        0.006
7            -2.2585766002E+00        3.457E-02        0.006
8            -2.2585544100E+00        7.833E-03        0.007
9            -2.2585535302E+00        6.738E-03        0.006
10           -2.2585531299E+00        1.447E-03        0.006
11           -2.2585534527E+00        3.481E-04        0.006
Total number of SCF: 11    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2585534527E+00 (Ha/atom)
Total free energy                  : -9.0342138108E+00 (Ha)
Band structure energy              :  6.9111688600E-01 (Ha)
Exchange correlation energy        : -4.4225557030E+00 (Ha)
Self and correction energy         : -1.2498741184E+01 (Ha)
-Entropy*kb*T                      : -3.6220584765E-03 (Ha)
Fermi level                        :  1.4538543174E-01 (Ha)
RMS force                          :  8.7070320830E-03 (Ha/Bohr)
Maximum force                      :  1.3913560403E-02 (Ha/Bohr)
Time for force calculation         :  0.014 (sec)
Pressure                           :  1.6502748559E+01 (GPa)
Maximum stress                     :  1.7029525887E+01 (GPa)
Time for stress calculation        :  0.026 (sec)
===================================================================
                    Self Consistent Field (SCF#9)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.3436771759E+00        5.590E-01        0.008
2            -2.2526929284E+00        2.839E-01        0.007
3            -2.2487945401E+00        1.891E-01        0.009
4            -2.2463187754E+00        9.937E-02        0.007
5            -2.2463637836E+00        1.045E-01        0.007
6            -2.2461278521E+00        5.630E-02        0.007
7            -2.2461193145E+00        5.607E-02        0.007
8            -2.2460503398E+00        1.090E-02        0.007
9            -2.2460475819E+00        7.564E-03        0.007
10           -2.2460524927E+00        1.892E-02        0.006
11           -2.2460466741E+00        1.214E-03        0.006
12           -2.2460468636E+00        1.233E-04        0.006
Total number of SCF: 12    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2460468636E+00 (Ha/atom)
Total free energy                  : -8.9841874542E+00 (Ha)
Band structure energy              :  7.0146069171E-01 (Ha)
Exchange correlation energy        : -4.5057886513E+00 (Ha)
Self and correction energy         : -1.2498677095E+01 (Ha)
-Entropy*kb*T                      : -2.7941318154E-03 (Ha)
Fermi level                        :  1.4804303210E-01 (Ha)
RMS force                          :  6.0332426474E-02 (Ha/Bohr)
Maximum force                      :  8.3882597426E-02 (Ha/Bohr)
Time for force calculation         :  0.013 (sec)
Pressure                           :  3.0618173487E+01 (GPa)
Maximum stress                     :  4.9703402576E+01 (GPa)
Time for stress calculation        :  0.022 (sec)
===================================================================
                    Self Consistent Field (SCF#10)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.3765775416E+00        6.365E-01        0.008
2            -2.2587421904E+00        2.671E-01        0.009
3            -2.2561238540E+00        1.854E-01        0.007
4            -2.2542304561E+00        1.685E-01        0.006
5            -2.2534636775E+00        9.526E-02        0.007
6            -2.2532908316E+00        4.372E-02        0.007
7            -2.2532788943E+00        3.698E-02        0.006
8            -2.2532587741E+00        1.980E-02        0.006
9            -2.2532525044E+00        9.334E-03        0.006
10           -2.2532509954E+00        3.766E-03        0.006
11           -2.2532506731E+00        1.228E-03        0.006
12           -2.2532508207E+00        3.117E-04        0.007
Total number of SCF: 12    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.2532508207E+00 (Ha/atom)
Total free energy                  : -9.0130032827E+00 (Ha)
Band structure energy              :  8.3393844712E-01 (Ha)
Exchange correlation energy        : -4.4953583259E+00 (Ha)
Self and correction energy         : -1.2498695411E+01 (Ha)
-Entropy*kb*T                      : -3.3559897952E-03 (Ha)
Fermi level                        :  1.6224381758E-01 (Ha)
RMS force                          :  1.5829696214E-02 (Ha/Bohr)
Maximum force                      :  2.1097281153E-02 (Ha/Bohr)
Time for force calculation         :  0.021 (sec)
Pressure                           :  2.5475634927E+01 (GPa)
Maximum stress                     :  4.6324288232E+01 (GPa)
Time for stress calculation        :  0.038 (sec)
